Configurability/ConfigurableServiceHelper.php

```diff
@@ -98,13 +96,11 @@ public function createContext(array $options, MessageInterface $message = null,
         return $context;
     }
 
-    public function resolveArray($input, &$context)
-    {
+    public function resolveArray($input, &$context){
```

Comment:
please run `php-cs-fixer fix vendor/smartbox/integration-framework-bundle/Configurability/ConfigurableServiceHelper.php --rules=@Symfony`.
The { should be on the next line.